packages/p/plf_hive/xmake.lua

```diff
@@ -0,0 +1,27 @@
+package("plf_hive")
+    set_kind("library", {headeronly = true})
+    set_homepage("https://plflib.org/colony.htm")
+    set_description("plf::hive is a fork of plf::colony to match the current C++ standards proposal.")
+    set_license("zlib")
+
+    add_urls("https://github.com/mattreecebentley/plf_hive.git")
+    add_versions("latest", "bebd22a4fa017c09bfa0100603e5c7ff4af6c01e")
```

Comment:
We use date for version, like this:

https://github.com/xmake-io/xmake-repo/blob/3a4a573cdfc1c6d86785c7d689ac02fc2863e4b9/packages/f/fast_io/xmake.lua#L11